test/crs/rand.jl

```diff
@@ -24,8 +24,10 @@
     @test rand(rng, GeodeticLatLon{WGS84Latest}) isa GeodeticLatLon
     @test rand(rng, GeodeticLatLon) isa GeodeticLatLon{WGS84Latest}
 
-    @test rand(rng, LatLon{WGS84Latest}) isa LatLon
-    @test rand(rng, LatLon) isa LatLon{WGS84Latest}
+    @test rand(rng, LatLon{WGS84Latest}) isa LatLon{WGS84Latest, Deg{Float64}}
+    @test rand(rng, LatLon) isa LatLon{WGS84Latest, Deg{Float64}}
+    @test rand(rng, LatLon{WGS84Latest}, 100) isa Vector{LatLon{WGS84Latest, Deg{Float64}}}
+    @test rand(rng, LatLon, 100) isa Vector{LatLon{WGS84Latest, Deg{Float64}}}
```

Comment:
The tests should only check the union-all type. The other type parameters are considered implementation detail.